include/deal.II/base/mpi.h

```diff
@@ -1463,6 +1485,36 @@ namespace Utilities
 #  endif
     }
 
+    template <typename T>
+    T
+    broadcast(const MPI_Comm &   comm,
+              const T &          object_to_send,
+              const unsigned int root_process)
+    {
+#  ifndef DEAL_II_WITH_MPI
+      (void)comm;
+      (void)root_process;
+      return object_to_send;
+#  else
+      const auto n_procs = dealii::Utilities::MPI::n_mpi_processes(comm);
+      AssertIndexRange(root_process, n_procs);
```

Comment:
You need to either put this in `#ifdef DEBUG` or write `(void)n_procs;`, otherwise a warning about an unused variable gets triggered in `RELEASE` mode.

Reply:
Thanks for the remark! I added `(void)n_procs;`